### DISCUSSION TITLE
Pricing plans: add "Inactive" users to the solo plan

### DISCUSSION THREAD

**Discussion by jan-dh** [craftcms/cms#14768] [General]
Wanted to open up this discussion. When looking at Craft CMS to build some sort of Saas tool it would make sense to enable the "Inactive" user approach for the solo plan as well. I mean, I could technically hack this functionality in by creating a user channel etc. but still. If users are only ment to be used as users who log in to your "product" and not in to the CMS, the $399 licence fee seems very steep.

**Comment by brandonkelly** [comment]
It’s not allowed because it doesn’t feel right to have multiple users (even if inactive), if they can’t be managed in the control panel.

We did just release Craft 5.1 which removes the requirement that all users must be admins. Still limited to 5 though.
